tests/ext-listen.test.js

```diff
@@ -47,17 +47,3 @@ test('startMonitor and stopMonitor should have their respective requestType', as
     requestType: 'stopMonitor',
   });
 });
-
-test('openActivityLogPage should create tab with ActivityLogPageURL', async () => {
```

Comment:
This has got coverage while testing the opening of Activity Logs page from browerAction popup (in `ext-popup.test.js`). Hence removing this.